### ISSUE TITLE
Which dataset is it in the demo?

### ISSUE THREAD

**Issue by hbk16** [zengyu714/segmentation-pytorch#1]
Which dataset is it in the demo? And where to download it? THX.

**Comment by zengyu714** [comment]
Here it is http://ijoint.istb.unibe.ch/challenge/index.html <http://ijoint.istb.unibe.ch/challenge/index.html>.
But this challenge seems not open currently.

You can just find another dataset to try some interesting algorithms.

**Comment by hbk16** [comment]
Thanks. 
I've noticed that labels in training are converted to type LongTensor, but wouldn't it save lots of memories if using ShortTensor or CharTensor instead in dice loss?

**Comment by zengyu714** [comment]
You’re right in some way.

Just when I wrote it, the loss function like `nll_loss <https://pytorch.org/docs/master/nn.html#nll-loss>` required specific LongTensor : )
I don’t know how it’s going now.
This example<https://github.com/pytorch/pytorch/blob/e8536c08a16b533fe0a9d645dd4255513f9f4fdd/torch/nn/functional.py#L1412> in the documentation says it might not be necessary at present. (Since the `target` would be `torch.float32`)

Anyway, just try more and choose the better one.
